VideoAPI/Video.API/Startup.cs

```diff
@@ -127,14 +129,14 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             }
 
             app.UseRouting();
-            
+
+            //app.UseMiddleware<LoggingMiddleware>();
```

Comment:
Would you mind deleting this if it's not required anymore?

Reply:
done